pkg/controller/service/controller_test.go

```diff
@@ -508,8 +508,8 @@ func TestGetNodeConditionPredicate(t *testing.T) {
 					},
 				},
 			},
-			expectAccept: false,
-			name:         "unschedulable",
+			expectAccept: true,
+			name:         "schedulable",
```

Comment:
Should the name have been changed?

Reply:
I changed it, but I can change it back.  The name is pretty terse anyway and should be longer.